Makefile

```diff
@@ -43,7 +43,7 @@ BUNDLE_GEN_FLAGS ?= -q --overwrite --version $(VERSION).0 $(BUNDLE_METADATA_OPTS
 OPERATOR_SDK_VERSION ?= v1.22.0-ocp
 
 # Image URL to use all building/pushing image targets
-IMG ?= quay.io/openshift/origin-ptp-operator:$(VERSION)
```

Comment:
This should still maintain $(version) since this will bemodified directly dowstream and never pull from upstream